modules/runner/src/integrationTest/java/org/apache/ignite/internal/table/ItInternalTableTest.java

```diff
@@ -384,78 +359,190 @@ public void testRoScanAllExplicitPopulatingData() throws InterruptedException {
         roScanAll(false);
     }
 
-    private void roScanAll(boolean implicit) throws InterruptedException {
-        IgniteImpl node = node();
+    @Test
+    public void getAllOrderTest() throws Exception {
+        var keyRows = populateEvenKeysAndPrepareEntriesToLookup(true);
```

Comment:
It is incorrect to use `var` because the type is not visible from the line of code.